src/ramses_tx/parsers.py

```diff
@@ -1378,22 +1378,23 @@ def parser_1fd4(payload: str, msg: Message) -> PayDictT._1FD4:
     return {"ticker": int(payload[2:], 16)}
 
 
-# WIP: unknown, HVAC
+# WIP: HVAC auto requests (confirmed for Orcon, others?)
 def parser_2210(payload: str, msg: Message) -> dict[str, Any]:
     try:
         assert msg.verb in (RP, I_) or payload == "00"
         assert payload[10:12] == payload[38:40] and payload[
             10:12
-        ] in (  # auto requested fan speed step?
+        ] in (  # auto requested fan speed %. Identical [38:40] for supply?
```

Comment:
This validation is too restrictive, In CO2 demand mode, this can be any percentage, the range is configurable between speed 1 and speed 3, and those speeds are also configurable.
Fan demand from the humidity sensor can also be configured, either to be speed 2 or speed 3.

Reply:
I took out the fixed values, kept an assert that `payload[10:12] == payload[38:40]`